### DISCUSSION TITLE
Version 2.4.0

### DISCUSSION THREAD

**Discussion by BourgeoisLab** [kleiner-brauhelfer/kleiner-brauhelfer-2#248] [General]
- Neu: Datenbankversion **2007**
- Neu: Negative SRE erlaubt
- Neu: Eigenes Fenster für Verwaltung der Wasserprofile
- Neu: Überprüfung der verstecken Eingabefelder bei ausgeschalteten Modulen
- Neu: Eingabefeld für Wassermenge für Verdünnung der Anstellwürze
- Neu: Hintergrundfarbe des Etiketts für jeden Sud einzeln einstellbar
- Neu: Umrechnungsformel für Brix nach Dichte von Petr Novotný (jens-brand)
- Fix: Reiter Ausrüstung erste ausgewählte Anlage
- Fix: diverse Fehler in der Brauübersicht korrigiert

<hr /><em>This discussion was created from the release <a href='https://github.com/kleiner-brauhelfer/kleiner-brauhelfer-2/releases/tag/v.2.4.0'>Version 2.4.0</a>.</em>

**Comment by Kajanax** [comment]
Wird es auch eine macos Version geben?

**Reply by realholgi** [reply]
Pfingstmontag, wenn ich aus dem Urlaub zurück bin…

**Reply by realholgi** [reply]
macOs version angehängt.

**Reply by bilzebub666** [reply]
Hallo,
ich habe versucht, die Mac-Version sowohl auf meinem MacBook Pro (Version 12.6.1), als auch auf meinem iPhone (6s Plus) und meinem iPad (Air) zu installieren, jedoch ging es leider auf keinem dieser Geräte. Woran könnte dies liegen? Ich würde auch sehr gerne mit dem Programm arbeiten.

Vielen Dank und HG

**Reply by bilzebub666** [reply]
P. S: Diese Fehlermeldung kommt:

„Kleiner Brauhelfer 2.app“ kann nicht geöffnet werden, da der Entwickler nicht verifiziert werden kann.
macOS kann nicht verifizieren, dass diese App keine Malware enthält.

**Reply by Rasinari** [reply]
Mit Maus auf Programm -> Rechte Maustaste Öffnen -> Fehlermeldung -> Rechte Maustaste Öffnen

**Comment by markussoeth** [comment]
Danke.
Wieder mal Klasse geworden!

**Reply by Rasinari** [reply]
Dem kann man sich nur anschließen!
